user/tests/integration/communication/events/events.cpp

```diff
@@ -4,28 +4,37 @@
 namespace {
 
 String eventName;
-String eventData;
+Variant eventData;
```

Comment:
Does it make sense to use EventData in the test to make sure the API doesn't break in the future?